### ISSUE TITLE
check XKB_DFLT_RULES_FILE

### ISSUE THREAD

**Issue by uli42** [ArcticaProject/nx-libs#371]
```
$ grep -r XKB_DFLT_RULES_FILE | grep define
xkb/xkbInit.c:#define	XKB_DFLT_RULES_FILE	"rules"
hw/nxagent/Keyboard.c:#define XKB_DFLT_RULES_FILE  "xfree86"
```

I think, this is definitely wrong somehow. Those two defines have the same name but point to different directory levels!

**Comment by uli42** [comment]
Ionic found https://cgit.freedesktop.org/xorg/xserver/commit/?id=463e02e7de5da3e582a3a049110a476713c7210e which I have backported.

**Comment by sunweaver** [comment]
Can #371 be closed with #372? If so, please add a closure to #372's commit message.
